### PR TITLE
Enhance FlushAll/GetFlushAllState API

### PR DESCRIPTION
**Pull request by wayblink**
issue: #26777

### CODE REVIEW

File: internal/datacoord/coordinator_broker.go [hunk 1]
```diff
@@ -34,7 +34,7 @@ const (
 )
 
 type Broker interface {
-	DescribeCollectionInternal(ctx context.Context, collectionID int64) (*milvuspb.DescribeCollectionResponse, error)
+	DescribeCollectionInternal(ctx context.Context, collectionID int64, db, collection string) (*milvuspb.DescribeCollectionResponse, error)
```
Comment (bigsheeper) [inline internal/datacoord/coordinator_broker.go:37]: how about wrapping a new interface?

Comment (wayblink) [reply internal/datacoord/coordinator_broker.go:37]: ok

File: internal/datacoord/errors.go [hunk 2]
```diff
@@ -45,6 +45,14 @@ func msgSegmentNotFound(segID UniqueID) string {
 	return fmt.Sprintf("failed to get segment %d", segID)
 }
 
+func msgDatabaseNotFound(dbName string) string {
```
Comment (bigsheeper) [inline internal/datacoord/errors.go:48]: using merr.ErrDataBaseNotFound instead is better

File: internal/datacoord/errors.go [hunk 3]
```diff
@@ -45,6 +45,14 @@ func msgSegmentNotFound(segID UniqueID) string {
...
+	return fmt.Sprintf("failed to get db %s", dbName)
+}
+
+func msgCollectionNotFound(collectionName string) string {
```
Comment (bigsheeper) [inline internal/datacoord/errors.go:52]: using merr.ErrCollectionNotFound is better

File: internal/proxy/impl.go [hunk 4]
```diff
@@ -2985,8 +2987,18 @@ func (node *Proxy) FlushAll(ctx context.Context, _ *milvuspb.FlushAllRequest) (*
 	if hasError(dbsRsp.GetStatus(), err) {
 		return resp, nil
 	}
+	dbNames := dbsRsp.DbNames
+	if req.GetDbName() != "" {
+		dbNames = lo.Filter(dbNames, func(dbName string, _ int) bool {
+			return dbName == req.GetDbName()
+		})
+		if len(dbNames) == 0 {
+			resp.Status.Reason = fmt.Sprintf("failed to get db %s", req.GetDbName())
```
Comment (bigsheeper) [inline internal/proxy/impl.go:2996]: ditto

**Pull request merged**